azurelinuxagent/common/osutil/default.py

```diff
@@ -269,50 +255,58 @@ def remove_legacy_firewall_rule(self, dst_ip):
                 "Unable to remove legacy firewall rule, won't try removing it again. Error: {0}".format(ustr(error)))
 
     def enable_firewall(self, dst_ip, uid):
+        """
+        It checks if every iptable rule exists and add them if not present. It returns a tuple(enable firewall success status, update rules flag)
+        enable firewall success status: Returns true if every rule check completed successfully otherwise false
```

Comment:
NIT: maybe change to - 
```suggestion
        enable firewall success status: Returns True if every firewall rule exists otherwise False
```